index.js

```diff
@@ -86,6 +88,53 @@ const meow = (helpText, options) => {
 
 	help = (description ? `\n  ${description}\n` : '') + (help ? `\n${help}\n` : '\n');
 
+	if (options.helpOptions) {
```

Comment:
There's a lot of logic here. Most of the logic here should be moved into one or more functions.

Reply:
Make sense. 👍